.ike-prow/test-keeper.yaml

```diff
@@ -0,0 +1,2 @@
+skip_validation_for: 
+  - 'cluster/'
```

Comment:
there is still also the original file https://github.com/dipak-pawar/ike-prow-plugins/blob/c32f52a822cedb073f21298dda3e98ecb424bf82/test-keeper.yaml - it should be removed (moved here)